packages/cli/package.json

```diff
@@ -61,7 +62,8 @@
     "prompts": "^2.4.2",
     "ramda": "^0.30.1",
     "rimraf": "^6.0.1",
-    "style-dictionary": "^4.0.1"
+    "style-dictionary": "^4.0.1",
+    "type-fest": "^4.26.1"
```

Comment:
What do we need this for?

Reply:
Good point, I thought I uninstalled it again 🙈